### PR TITLE
Fixes Issue #388 Added better image in third intro of Circle of Trust

### PR DESCRIPTION
**Pull request by madhavrathi**
New Look :- 
new

### CODE REVIEW

**Comment by madhavrathi** [comment]
@sandarumk  Please review this.

File: app/app.iml [hunk 1]
```diff
@@ -76,39 +76,49 @@
       <sourceFolder url="file://$MODULE_DIR$/src/test/jni" isTestSource="true" />
```
Comment (sandarumk) [inline app/app.iml]: This should not be in version control

**Review by sandarumk — CHANGES_REQUESTED** [review]

File: app/app.iml [hunk 2]
```diff
@@ -73,42 +73,51 @@
       <sourceFolder url="file://$MODULE_DIR$/src/test/assets" type="java-test-resource" />
```
Comment (sandarumk) [inline app/app.iml]: It is still here. Please remove app.iml file.

**Review by sandarumk — CHANGES_REQUESTED** [review]

File: app/app.iml (hunk 2 above)
Comment (madhavrathi) [reply app/app.iml]: Please help me out here. How do i remove app.iml file from my commit ?

Comment (sandarumk) [reply app/app.iml]: This should do the trick. Next time when committing make sure not to add it. http://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo
If you face a problem, a quick google search should help. There are plenty of related questions in Stack Overflow. Once you add the commit, try to squash your commits so it should appear as one. If not at least remove the app.iml fie then I will squash your commits when merging.

**Comment by sandarumk** [comment]
Did you remove the existing low quality image?

**Comment by madhavrathi** [comment]
@sandarumk Not yet, should I ?

**Comment by sandarumk** [comment]
yes.

**Comment by madhavrathi** [comment]
@sandarumk i have now made the changes  into single commit. Sorry for the Delay

**Comment by sandarumk** [comment]
Sorry I forgot this earlier.
https://developer.android.com/guide/practices/screens_support.html

**Comment by madhavrathi** [comment]
@sandarumk  i didn't get your point

**Comment by sandarumk** [comment]
Did you read the article on the link?

**Pull request closed without merging**